build/ci/e2e/custom_operator.jenkinsfile

```diff
@@ -0,0 +1,53 @@
+pipeline {
```

Comment:
Maybe custom_operator_image to be more explicit? `custom operator` sounds like "bring your own operator implementation".

Reply:
Fixed